run/archiveloop

```diff
@@ -227,7 +227,17 @@ function wait_for_archive_to_be_unreachable () {
 }
 
 function check_if_usb_gadget_is_mounted () {
-    if [ ! -d /sys/devices/platform/soc/??980000.usb/gadget/lun0 ]
+    local found="false"
+    for lun in /sys/devices/platform/soc/??980000.usb/gadget/lun0
+    do
+      if [ -d "$lun" ]
+      then
+        found="true"
+        break
+      fi
+    done
+
+    if [ "$found" = "false" ]
```

Comment:
It was shellcheck. 

The error related to `[ -d <file> ]` failing if the pattern matched more than one file. I was trying to limit the number of 'disable' lines. I can go either way on this one because the code grows quite a bit.